src/ui/VideoArea.qml

```diff
@@ -265,6 +265,7 @@ Item {
                 onCurrentFrameChanged: {
                     fovChanged();
                     controller.update_keyframe_values(timestamp);
+                    window.motionData.orientationIndicator.updateOrientation(timeline.position * timeline.durationMs * 1000)

```

Comment:
isn't `timeline.position * timeline.durationMs * 1000` the same as just `timestamp`?

Reply:
Oh oops, just `timestamp*1000` works the same. Somehow missed that one